mlir/test/CAPI/smt.c

```diff
@@ -0,0 +1,186 @@
+
+/*===- smtlib.c - Simple test of SMTLIB C API -----------------------------===*\
+|*                                                                            *|
+|* Part of the LLVM Project, under the Apache License v2.0 with LLVM          *|
+|* Exceptions.                                                                *|
+|* See https://llvm.org/LICENSE.txt for license information.                  *|
+|* SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception                    *|
+|*                                                                            *|
+\*===----------------------------------------------------------------------===*/
```

Comment:
Nit: the filename doesn't match and the way it is commented out is not really consistent with the other files in this directory